src/toil/test/utils/utilsTest.py

```diff
@@ -300,8 +300,8 @@ def testGetPIDStatus(self):
         wf.wait()
         self.check_status('COMPLETED', status_fn=ToilStatus.getPIDStatus)
 
-        # TODO: we need to reach into the FileJobStore's files and
-        # delete this shared file. We assume we know its internal layout.
+        # TODO: we need to reach into the FileJobStore's files and delete this
+        #  shared file. We assume we know its internal layout.
         os.remove(os.path.join(self.toilDir, 'files/shared/pid.log'))
```

Comment:
We need a `delete_shared_file()` method in the job store to refactor this.

Reply:
This makes sense to leave as is; we can fix the test hack if we ever do add that method.